### ISSUE TITLE
Please avoid wasting space on the sides

### ISSUE THREAD

**Issue by Lastique** [boostorg/website-v2#936]
On the new website, the actual content takes about 2/3 of the screen width, leaving huge empty bars on the sides and condensing the content in the middle. This is the case both for the main website content, as well as the library documentation. For example, on a 3840x2160 screen, the main website content is only about 2430 pixels wide. Library documentation is about 2546 pixels wide.

This reduces the amount of information that fits on the screen and requires more scrolling. Please make the content wider.

A few reference points:

- [GitHub](https://github.com/) website: full width. When [viewing a repository](https://github.com/cppalliance/temp-site), main content: >75% width, header content spans full width
- [StackOverflow](https://stackoverflow.com/): main content: >75% width, with some notifications (e.g. about cookies) presented on the side
- [GitLab](https://gitlab.com/) website: full width.
- [Godbolt](https://godbolt.org/): full width.
- [isocpp.org](https://isocpp.org/): main content on the front page: ~73% width, with a navigation panel on the left. Other pages (e.g. [here](https://isocpp.org/std/status)): full width.
- [Current Boost website](https://www.boost.org/): full width.

I think, main content should take at least 75-80% of the screen width, preferably full width.

**Comment by vinniefalco** [comment]
We tried it that way but the problem is that the pages don't look good when all the text is on a single line, which is what happens when you make the window wide enough horizontally. At the end of the day the physical presentation is an editorial decision driven by aesthetics and opinions. And everyone is going to have different opinions. What we have developed attempts to strike a precise balance between form and function.

**Comment by Lastique** [comment]
I'm quite satisfied how the text looks on the current website.

I suspect, the problems you had were at least in part caused by the font used on the new website. The old website used a more legible and less condensed font.

**Comment by vinniefalco** [comment]
Every site you linked has a maximum page size, ours is no different except for godbolt.org. But that website does not show any exposition. And by the way, I raised the same point you have raised about wasting space, to our front-end designer and our artist. I saw what the resulting page looked like at 3,000+ pixels width, and it was not good at all. The problem with unrestricted horizontal stretching is that you end up with every paragraph on its own line which is the width of the screen. It should be obvious that this is very poor both aesthetically and in terms of functionality. The "page" needs a maximum width to be designed nicely. Some Boost documentation (including mine) has unrestricted horizontal sizing and the result is terrible:

![image](https://github.com/cppalliance/temp-site/assets/1503976/d2d99827-ac2e-40af-8e0e-9a894ab8455e)

This does not look good to me, or other people that I consulted such as @pdimov. Even his documentation has maximum page size:

![image](https://github.com/cppalliance/temp-site/assets/1503976/55f38957-2864-4d57-9f0c-a55580604663)

I do not think that we should be pioneers here; maximum page widths exist for a reason. We can tweak some flows like the header or table of contents but I do not think that we should be designing for unlimited horizontal stretching across the entire site.
